### PR TITLE
machines: Always check the 'Automatically start libvirt on boot'

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
Some debugging notes:

Looks like https://github.com/cockpit-project/cockpit/pull/10284/commits/7ca67bebd34d311b8f78d3f195e0e296abab3eb3 solved the issue, so it's seem to be some race condition here. 
Note that UPDATE_LIBVIRT_STATE is updated three times,
1. Within INIT_DATA_RETRIEVAL
2. Within GET_ALL_VMS
3.Within the event listener which catches the signal that libvirtd was stopped. And this happens exactly after the page was loaded.
These are all done asynchronously so I wouldn't be surprised that global state updates somehow didn't get the correct order.

**Comment by KKoukiou** [comment]
testLibvirt now failed later in the code, have to recheck.

File: pkg/machines/components/libvirtSlate.jsx [hunk 1]
```diff
@@ -107,7 +85,7 @@ class LibvirtSlate extends React.Component {
                         <input type="checkbox"
                                id="enable-libvirt"
                                disabled={!name}
-                               checked={this.state.libvirtEnabled}
+                               checked
```
Comment (martinpitt) [inline pkg/machines/components/libvirtSlate.jsx]: This looks a bit curious. As long as long as there still actually is a checkbox, the component should have state for it and set it here. Just the default state should change.

Comment (KKoukiou) [reply pkg/machines/components/libvirtSlate.jsx]: Ups, good catch. Removed that by mistake.

Comment (KKoukiou) [reply pkg/machines/components/libvirtSlate.jsx]: Ups, indeed. I removed it by mistake.

**Pull request merged**